src/engine/renderer/tr_world.cpp

```diff
@@ -1061,10 +1036,8 @@ void R_AddPrecachedWorldInteractions( trRefLight_t *light )
 				{
 					continue;
 				}
-				else
-				{
```

Comment:
A lot of stuff here, for example this, or changing `if (p == nullptr)` to `if (!p)`,  is just cosmetic changes which don't change the meaning of the code.